usecase/model/ibc/msg_recv_packet.go

```diff
@@ -33,5 +33,6 @@ type MsgRecvPacketFungibleTokenDenominationTrace struct {
 }
 
 type MsgRecvPacketPacketAck struct {
-	Result []byte `mspstructure:"result" json:"result"`
+	MaybeResult []byte `mapstructure:"result" json:"result"`
+	MaybeError  string `mapstructure:"error" json:"error"`
```

Comment:
If it is `Maybe*`, it is expected to be a nil-able value. So should use `*string` here.

Reply:
Got it! Let me change this.